src/DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -494,6 +494,17 @@ function PreFirstEntryMapFixes()
     //#region Free Clinic
     case "02_NYC_FREECLINIC":
         SetAllLampsState(true, true, false); // the free clinic has one desk lamp, at a desk no one is using
+
+        // the free clinic shares a KeyID with the MJ12 facility beneath unatco
+        foreach AllActors(class'NanoKey', k) {
+            if (k.KeyID == 'Cabinet') {
+                k.KeyID = 'MedCabDoor';
+            }
+        }
+        foreach AllActors(class'DeusExMover', d, 'MedCabDoor') {
```

Comment:
this is more important than most changes, because a mismatch of the key and the door will cause the player to be unable to open the door

```suggestion
        foreach AllActors(class'#var(DeusExPrefix)Mover', d, 'MedCabDoor') {
```